src/main/java/com/accantosystems/stratoss/vnfmdriver/web/etsi/ETSIExceptionHandlingControllerAdvice.java

```diff
@@ -90,8 +90,8 @@ protected ProblemDetails handleNotImplementedException(HttpServletRequest req, N
     @ExceptionHandler(ResponseTypeNotAcceptableException.class)
     @ResponseStatus(HttpStatus.NOT_ACCEPTABLE)
     @ResponseBody
-    protected ProblemDetails handleResponseTypeNotAcceptableException(HttpServletRequest req, ResponseTypeNotAcceptableException cause) {
-        return defaultHandle("The requested response type was not acceptable", cause, HttpStatus.NOT_ACCEPTABLE);
+    protected ResponseEntity<String> handleResponseTypeNotAcceptableException(HttpServletRequest req, ResponseTypeNotAcceptableException cause) {
+        return new ResponseEntity<>("The requested response type was not acceptable", HttpStatus.NOT_ACCEPTABLE);
```

Comment:
sorry but I don't think this is the right fix. We can't just change the response type from a ProblemDetails structure (as expected by the ETSI spec) to plain string response, especially as it won't contain the cause as it did previously. Also by not calling defaultHandle we no longer have any of the logging that went on in that method